### ISSUE TITLE
Make clear that track emails do not come from requesters mentioned therein

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/alaveteli#7307]
On WDTK, we have recently had at least two instances where users have replied to a track (therefore to the support inbox), but addressing their reply to the user whose requests had triggered the alert.  In both of these recent cases that I dealt with myself, the users had been following a particular requester, and had no other tracks.

We could perhaps see if we can change the wording to fix this.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
